bundles/org.openhab.binding.wiz/src/main/java/org/openhab/binding/wiz/internal/config/WizDeviceConfiguration.java

```diff
@@ -0,0 +1,36 @@
+/**
+ * Copyright (c) 2010-2024 Contributors to the openHAB project
+ *
+ * See the NOTICE file(s) distributed with this work for additional
+ * information.
+ *
+ * This program and the accompanying materials are made available under the
+ * terms of the Eclipse Public License 2.0 which is available at
+ * http://www.eclipse.org/legal/epl-2.0
+ *
+ * SPDX-License-Identifier: EPL-2.0
+ */
+package org.openhab.binding.wiz.internal.config;
+
+import static org.openhab.binding.wiz.internal.WizBindingConstants.*;
+
+import org.eclipse.jdt.annotation.NonNullByDefault;
+
+/**
+ * The {@link BondHomeConfiguration} class contains fields mapping thing
+ * configuration parameters.
+ *
+ * @author Sara Geleskie Damiano - Initial contribution
+ */
+@NonNullByDefault
+public class WizDeviceConfiguration {
+
+    /**
+     * Configuration for a Bond Bridge or Device
+     */
+    public String bulbMacAddress = MISSING_INVALID_MAC_ADDRESS;
+    public String bulbIpAddress = MISSING_INVALID_IP_ADDRESS;
```

Comment:
What if the hostname for my bulb is `bulbIpAddress`? 🙂 Perhaps this would be better?
```suggestion
    public String bulbIpAddress = "";
```

(with corresponding change in validation)